test/unit/org/apache/cassandra/cql3/statements/DescribeStatementTest.java

```diff
@@ -477,25 +477,37 @@ public void testDescribeTableWithInternals() throws Throwable
                                       "    v3 int,\n" +
                                       "    PRIMARY KEY ((pk1, pk2), c)\n" +
                                       ") WITH ID = " + id + "\n" +
```

Comment:
This test class is not present in DSE, so these changes are to adjust the new dropped column syntax.